kubespawner/spawner.py

```diff
@@ -803,12 +803,35 @@ def _hub_connect_port_default(self):
 
     profile_form_template = Unicode(
         """
-        <label for="profile">Please select a profile to launch</label>
-        <select class="form-control" name="profile" required autofocus>
-            {% for profile in profile_list %}
-            <option {% if profile.default %}selected{% endif %} value="{{ loop.index0 }}">{{ profile.display_name }}</option>
-            {% endfor %}
-        </select>
+        <script>
+        // JupyterHub 0.8 applied form-control indisciminately to all form elements.
+        // Can be removed once we stop supporting JupyterHub 0.8
+        $(document).ready(function() {
+            $('#kubespawner-profiles-list input[type="radio"]').removeClass('form-control');
+        });
+        </script>
+        <style>
+        /* The profile description should not be bold, even though it is inside the <label> tag */
+        #kubespawner-profiles-list label p {
+            font-weight: normal;
+        }
+        </style>
+
+        <div class='form-group' id='kubespawner-profiles-list'>
+        {% for profile in profile_list %}
+        <label for='profile-item-{{ loop.index0 }}' class='form-control input-group'>
```

Comment:
I don't thin it makes sense for the item to be labeled to be *inside* its own label. Maybe the label should only be around the `profile.display_name`? I think that's how it usually works, and would also remove the need to revert and un-revert the bold styling, which you have here with css and strong tags.

Reply:
Something like:

```html
<div class='col-md-11'>
<label for="profile-item-{loop.index0}">{{profile.display_name}}</label>
<p>description...
```

Reply:
The label is parent of everything since I want users to be able to click anywhere on it and select the radio button. The only way I've found to do this without even more JS is to put them all in the label.

Reply:
target input inside `label` is valid according to MDN. There is just a warning that having links _within_ the label can be an accessibility concern as it will make it hard to activate the form element.

Seeing that profile descriptions can heave links concerns me, isn't jinja supposed to auto escape  values unless marked `|safe` ?

Reply:
Good point. I don't think that's jinja's default behavior, so we should probably opt-in to autoescape=True here to be safe.